pom.xml

```diff
@@ -171,7 +192,8 @@
             </goals>
             <configuration>
               <executable>${basedir}/exclude-files.sh</executable>
-              <commandlineArgs>${buildDirectory}/${project.artifactId}-${project.version}</commandlineArgs>
+              <commandlineArgs>${basedir}/dependency-excludes.txt
```

Comment:
Isn't the command line argument supposed to be?

```code
<>${basedir}/dependency-excludes-prod.txt<>

Reply:
It is not. It's only for the prod profile that we process the **dependency-excludes-prod.txt** file.